ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot

```diff
@@ -55,11 +56,12 @@ Verify Pod Toleration
 
 Verify RHODS User Groups
     [Documentation]    Verify User Configuration after the upgrade
-    [Tags]      Upgrade
-    ${admin}        Set Variable        ${payload[0]['spec']['groupsConfig']['adminGroups']}
-    ${user}     Set Variable        ${payload[0]['spec']['groupsConfig']['allowedGroups']}
-    Should Be Equal As Strings      '${admin}'      'rhods-admins,rhods-users'
```

Comment:
What about check for the rhods-users? This group is still assigned as admins in the pre-upgrade test suite.
https://github.com/red-hat-data-services/ods-ci/blob/cb949633d30b38a7a524219aa5f611770fcd52a9/ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot#L78

Reply:
that group is not used by default https://github.com/opendatahub-io/opendatahub-operator/blob/main/controllers/components/dashboard/dashboard_support.go#L34 @MarianMacik 
I can update the pre-upgrade test suite on another PR

Reply:
@asanzgom but that group is explicitly added in the pre-upgrade test suite, so will it actually work? We are explicitly setting these 2 groups as administrators in pre-upgrade test suite, so in post-upgrade we should check both groups are present, right?

Reply:
You are right @MarianMacik I will add it too